### PR TITLE
created the nav bar

### PR DESCRIPTION
**Pull request by navya-shajan**
navigation bar with menus and search bar.

@MrTim

@keponk
@Nadezhda
@Golovnev

### CODE REVIEW

**Comment by keponk** [comment]
hi @navya-shajan this is great! How does it feel working on this? Was having the wireframes any use? And is there any part that you feel is hard to move forward?

The code looks great as well. For the sake of code cleanup I would suggest you delete the empty lines between the code. You can do this manually or let your IDE do it for you. ;)

File: movieApp/styles.css [hunk 1]
```diff
@@ -1,17 +1,55 @@
...
+}
 
-footer{
-    background-color: black;
-    width: max-content;
-    height: 100px;
+label.logoText{
+    color:white ;
+    font-size: 10px;
+
```
Comment (keponk) [inline movieApp/styles.css]: no need for these empty lines

File: movieApp/index.html [hunk 2]
```diff
@@ -3,25 +3,40 @@
...
-    <header id="banner" class="body">
-    <h1><a href="#">Movie App </a></h1>
-  
-    <nav><ul class="topNavBar">
-      <li><a href="#">HOME</a></li>
-      <li><a href="#">GENRES</a></li>
-  
-      <li><a href="#">KIDS</a></li>
-      
-    </ul></nav>
```
Comment (keponk) [inline movieApp/index.html]: many empty lines here

File: movieApp/index.html [hunk 3]
```diff
@@ -3,25 +3,33 @@
...
-</html>
\ No newline at end of file
+    <header>
+      <nav>
+        <label class="logo">MFA</label>
+        <label class="logoText">Movie Finder App</label>   
+        <ul>
+          <li><a href="#">HOME</a></li>
+          <li><a href="#">GENRES</a>
+            <!--  <ul class="subMenu">
```
Comment (MrTim) [inline movieApp/index.html:17]: Suggestion for future: don't leave commented code in something which you are going to put for review and then merge to master. 
For now it's fine, just a hint :)

File: movieApp/styles.css [hunk 4]
```diff
@@ -1,17 +1,53 @@
+*{
```
Comment (MrTim) [inline movieApp/styles.css:1]: put a space before `{`.

File: movieApp/styles.css [hunk 5]
```diff
@@ -1,17 +1,53 @@
...
-#banner{
-height: 500px;
- background-color: rgb(168, 145, 145); 
- text-align: center;
-}
\ No newline at end of file
+label.logoText{
+    color:white ;
+    font-size: 10px;
+    }
```
Comment (MrTim) [inline movieApp/styles.css:29]: pay attention for indentation. As well you can install [Prettier plugin](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode) for VSCode and he'll take care of those things!

**Review by MrTim — APPROVED** [review]
:+1: Just left some comments and suggestions for future.

**Pull request merged**